src/puppetlabs/puppetdb/status.clj

```diff
@@ -0,0 +1,39 @@
+(ns puppetlabs.puppetdb.status
+  (:require [puppetlabs.puppetdb.scf.storage-utils :as sutils]
+            [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.puppetdb.mq :as mq]
+            [puppetlabs.puppetdb.schema :as pls]
+            [schema.core :as s]))
+
+(def status-details-schema {:maintenance_mode? s/Bool
```

Comment:
Or if our plan is to duplicate some of the stuff in this whole namespace, maybe calling it pdb_status.clj like we have with pdb_routing.clj and naming the pe namespace pe_status.clj (like pe_routing.clj).